mail-client/alot/alot-0.8.1.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
Update to `EAPI=7`.
You will need to remove `vcs-snapshot` from the `inherit`

I also suggest to update the `SRC_URI` to:
`SRC_URI="https://github.com/pazz/alot/archive/${PV}.tar.gz -> ${P}.tar.gz"`

Reply:
upgraded